birthdays/locales/de.po

```diff
@@ -0,0 +1,59 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR ORGANIZATION
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: 2018-05-22 22:28-0400\n"
+"PO-Revision-Date: 2018-05-23 18:26+0200\n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: pygettext.py 1.5\n"
+"X-Generator: Poedit 2.0.7\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: de_DE\n"
+
+#: birthdays.py:224
+msgid "Birthday List"
+msgstr "Geburtstagsliste"
+
+#: birthdays.py:229
+msgid "<@!{}> is now **{} years old**. :tada:"
+msgstr "<@!{}> ist nunt **{} Jahre alt**. :tada:"
+
+#: birthdays.py:230
+msgid "It's <@!{}>'s birthday today! :tada:"
+msgstr "Es ist <@!{]>s Geburtstag heute! :tada:"
+
+# Translation may require some small reworking as I do not know what the exact output is suppose to look like - Sentry
+#: birthdays.py:231
+msgid ""
+":white_check_mark: The birthday role on **{g}** has been set to: **{r}**."
+msgstr ""
+":white_check_mark: The Geburtstags Rolle am **{g}** wurde auf: **{r}** "
+"gesetzt."
+
+#: birthdays.py:232
+msgid ":x: The birthday date you entered is invalid. It must be `MM-DD`."
+msgstr ":x: Das eingegebene Geburtsdatum ist ungültig. Es muss `MM-DD` sein."
+
+# Translation may be inaccurate as I do not know what the exact output is suppose to look like - Sentry
```

Comment:
{} should be replaced with the locale's month word + day (not necessarily in that order). Example for english: September 15